dask_geomodeling/raster/sources.py

```diff
@@ -333,6 +336,10 @@ def gdal_dataset(self):
             self._gdal_dataset = gdal.Open(path)
             return self._gdal_dataset
 
+    def close_dataset(self):
```

Comment:
I don't know how this class is used. Would it be handy to use it as a context manager? Dunno.

Reply:
Normally you don't need to close the file handle, because if you do `del block`, the reference to the GDAL dataset is gone and it closes the file handle by itself. That is a design decision done by the GDAL people.

Only in the unittest I stick the block to the test class, so that the file handle is still open when I try to cleanup the tiff file, which fails on windows.